.env.sandbox.template

```
@@ -0,0 +1,78 @@
+# --- S4 Blockchain & Custody (Fireblocks — optional) ---
+# Request sandbox: https://www.fireblocks.com/developer-sandbox-sign-up/
+# Download RSA private key PEM file from Fireblocks console
+FIREBLOCKS_SANDBOX_API_KEY=your_fireblocks_api_key_id
+FIREBLOCKS_SANDBOX_API_SECRET_PATH=/path/to/fireblocks_secret.pem
+FIREBLOCKS_SANDBOX_VAULT_ACCOUNT_ID=0
+
+# --- S2 Compliance (Persona KYC — recommended, free) ---
+# Signup: https://app.withpersona.com/sign-up
+# Get sandbox API key from: Dashboard > Settings > API Keys
+PERSONA_SANDBOX_API_KEY=persona_sandbox_your_key_here
+PERSONA_INQUIRY_TEMPLATE_ID=itmpl_your_template_id_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PERSONA_INQUIRY_TEMPLATE_ID key should go before the PERSONA_SANDBOX_API_KEY key

(UnorderedKey)